doc/WALLET_DESKTOP.md

```diff
@@ -0,0 +1,104 @@
+# Wallet Desktop
```

Comment:
Maybe we can have another title here (and change the file name as well). To make it clear the tutorial is about using ledger.

Reply:
Changed the title, but not the file name.